pkg/cache/cache_test.go

```diff
@@ -61,8 +60,9 @@ var _ = Describe("Nodes", func() {
 
 				Expect(errors.Is(err, randomError)).To(BeTrue())
 			},
-			Entry("force=true", true),
-			Entry("force=false", false),
+			EntryDescription("force=%t"),
```

Comment:
Nice, didn't know that.

Reply:
Yes, not sure if this was already present in v1 or if it's new.